Doc/library/ctypes.rst

```diff
@@ -1799,10 +1799,32 @@ different ways, depending on the type and number of the parameters in the call:
    integer. *name* is name of the COM method. *iid* is an optional pointer to
    the interface identifier which is used in extended error reporting.
 
+   If *iid* is not specified, a :exc:`WindowsError` is raised if the COM method
+   call fails. If *iid* is specified, a :exc:`COMError` is raised instead.
```

Comment:
Err... I wonder why the reference is not picked up here, because the class *is* documented. (May be a Sphinx issue?)

Reply:
It has become harder to see after the rebase, but in the earlier [commit without the dot](https://github.com/python/cpython/commit/371110d4959b399ec87b859365778b131999b51b), "Tests / Docs / Docs (pull_request)" failed in CI.
In contrast, the [commit with the dot](https://github.com/python/cpython/commit/58cc032eb75a08ad64ea58317ce2ef93bc5fc34c) succeeded. 

Additionally, without the dot, no link was generated.

I think also it's probably an issue with Sphinx.

Reply:
> Additionally, without the dot, no link was generated.

That's because it couldn't find the reference. I'm wondering whether it couldn't find the reference because the exception is documented after or because of how it's declared (it might also happen that the module's scope was changed somewhere)